src/test/java/gov/healthit/chpl/aqa/stepDefinitions/CMSIDsLookupSteps.java

```diff
@@ -0,0 +1,98 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+import static org.testng.Assert.assertTrue;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import gov.healthit.chpl.aqa.pageObjects.CMSidReverseLookupPage;
+import gov.healthit.chpl.aqa.pageObjects.SearchPage;
+
+/**
+ * Class CMSIDsLookupSteps definition.
+ */
+public class CMSIDsLookupSteps {
+
+    private WebDriver driver;
+    private static final int DELAY = 40;
+        
+    public CMSIDsLookupSteps() {
+    driver = Hooks.getDriver();
+    }
+    
+    /**
+     * To load CMS ID Reverse Lookup page 
+     */
+    
+    @Given("^I am an end-user on CMS ID Reverse Lookup page$")
+    public void i_am_an_end_user_on_CMS_ID_Reverse_Lookup_page() {
+        driver.get("https://chpl.ahrqstg.org/#/resources/cms_lookup");
```

Comment:
Please look at the SED Steps and implement the part with the url that's brought in from the command line